src/App.jsx

```diff
@@ -3,6 +3,7 @@ import Nav from "./components/Nav/Nav"
 import HomePage from "./components/HomePage/HomePage"
 import "./App.css"
 import ClaimedBooks from "./Components/HomePage/ClaimedBooks/ClaimedBooks"
+import AvailableBooks from "./components/HomePage/AllBooks/AvailableBooks/AvailableBooks"
 
```

Comment:
Perhaps move the import "./App.css" below the components imports, to separate the import blocks.